app/lib/actions/katello/content_view/publish.rb

```diff
@@ -70,35 +71,9 @@ def humanized_name
         end
 
         def run
-          view = ::Katello::ContentView.find(input[:content_view_id])
-          version = ::Katello::ContentViewVersion.find(input[:content_view_version_id])
-          output[:content_view_id] = view.id
```

Comment:
what would the reason be to take values from the input and put them into the output? seems unnecessary

Reply:
agreed, i think it is unnecessary